func3.py

```diff
@@ -0,0 +1,15 @@
+# Write a function that finds the fibonacci sequence up to n elements iteratively and another one recursively.
+
+def fib(n):    
+    a, b ,i = 1, 1, 1
+    while i <= n:
+        print(a)
+        a, b = b, a+b
+        i=i+1
```

Comment:
Since you don't actually need the `i` you could replace it with a throwaway variable and use a `for` like this `for _ in range(1, n+1):`